src/sickle.ts

```diff
@@ -129,6 +129,8 @@ class Annotator {
         this.writeTextFromOffset(writeOffset, fnDecl.body);
         this.visit(fnDecl.body);
         break;
+      case ts.SyntaxKind.TypeAliasDeclaration:
+        this.visitTypeAlias(<ts.TypeAliasDeclaration>node);
```

Comment:
Nice catch!  Wow, that's what I get for writing a lot of Go code...

(The reason this even worked is that "writeNode" previously did nothing for "type" statements so the fallthrough didn't break anything.  I fixed it though.)